recipes-containers/pantavisor/pv-pvr-sdk_v1.2.2.bb

```diff
@@ -0,0 +1,10 @@
+SUMMARY = "Pantahub Apps hosted packages"
+LICENSE = "CLOSED"
```

Comment:
lets check licensing approach here... its definitely not CLOSED :)... of course my fault using this shortcut.